hr_family/models/hr_children.py

```diff
@@ -0,0 +1,31 @@
+# -*- coding:utf-8 -*-
+#
+#
+#    Copyright (C) 2011,2013 Michael Telahun Makonnen <mmakonnen@gmail.com>.
+#    All Rights Reserved.
+#
+#    This program is free software: you can redistribute it and/or modify
+#    it under the terms of the GNU Affero General Public License as published
+#    by the Free Software Foundation, either version 3 of the License, or
+#    (at your option) any later version.
+#
+#    This program is distributed in the hope that it will be useful,
+#    but WITHOUT ANY WARRANTY; without even the implied warranty of
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU Affero General Public License for more details.
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+#
+
+from openerp import models, fields
+
+
+class hr_children(models.Model):
+    _name = 'hr.employee.children'
+    _description = 'HR Employee Children'
+
+    name = fields.Char("Name", size=256, required=True)
```

Comment:
No particular reason, it was declared like that in the branch 7.0 (https://github.com/OCA/hr/blob/7.0/hr_family/hr.py). I will drop all ``size=`` from these fields.